assets/src/common/helpers/index.js

```diff
@@ -20,18 +20,13 @@ import {
 	VIDEO_ALLOWED_MEGABYTES_PER_SECOND,
 } from '../constants';
 
-// @todo This is is not guaranteed to exist.
-const { ampStoriesEditorSettings = {} } = window;
-const { allowedVideoMimeTypes = [ 'video/mp4' ] } = ampStoriesEditorSettings;
+const allowedVideoMimeTypes = [ 'video/mp4' ];
```

Comment:
If `ampStoriesEditorSettings` was set the value here would be `[ 'video/mp4', 'video/webm ]`. Is that still the case?

Reply:
No, like this should be fine. I don't think this is actually used outside the stories editor. So perhaps we need some follow-up refactoring to remove that variable.

Reply:
Actually, it can be removed, and the reference to it in `isFileTypeAllowed` as well.